test/src/sdk-harness/test_projects/storage_vec/u8/mod.rs

```diff
@@ -0,0 +1,163 @@
+mod utils;
+
+use utils::{
+    setup::get_contract_instance,
+    wrappers::{clear, get, insert, is_empty, len, pop, push, remove, swap_remove},
+};
+
+mod success {
+    use super::*;
+
+    #[tokio::test]
+    async fn can_get_contract_id() {
+        let (_instance, _id) = get_contract_instance().await;
+    }
```

Comment:
This is not a test. It fetches the instance but it does not test anything. Could you remove it?